api/v1alpha1/vaultdynamicsecret_types.go

```diff
@@ -58,8 +58,10 @@ type VaultDynamicSecretSpec struct {
 
 // VaultDynamicSecretStatus defines the observed state of VaultDynamicSecret
 type VaultDynamicSecretStatus struct {
-	// LastRenewalTime of the last, successful, secret lease renewal,
+	// LastRenewalTime of the last successful secret lease renewal,
```

Comment:
Looks like you missed one extra comma.. Reminder to run `make generate manifests` after this fix, I'll skip comments related to this during the review.

```suggestion
	// LastRenewalTime of the last successful secret lease renewal.
```